src/toil_lib/tools/variant_manipulation.py

```diff
@@ -0,0 +1,351 @@
+    command = ['-T', 'ApplyRecalibration',
+               '-mode', mode,
+               '-R', 'genome.fa',
+               '-input', 'input.vcf',
+               '-o', 'vqsr.vcf',
+               '-ts_filter_level', '99.0',  # Filter using second lowest tranche sensitivity
+               '-recalFile', 'recal',
+               '-tranchesFile', 'tranches']
+
+    if unsafe_mode:
+        command.extend(['-U', 'ALLOW_SEQ_DICT_INCOMPATIBILITY'])
+
+    docker_call(work_dir=work_dir,
+                env={'JAVA_OPTS': '-Djava.io.tmpdir=/data/ -Xmx{}'.format(job.memory)},
+                parameters=command,
+                tool='quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
+                inputs=inputs.keys(),
+                outputs={'vqsr.vcf': None})
+
+    return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'vqsr.vcf'))
+
+
+def gatk_combine_variants(job, vcfs, ref_fasta, ref_fai, ref_dict, merge_option='UNIQUIFY'):
+    """
+    Merges VCF files using GATK CombineVariants
+
+    :param JobFunctionWrappingJob job: Toil Job instance
+    :param dict vcfs: Dictionary of VCF FileStoreIDs {sample identifier: FileStoreID}
+    :param str ref_fasta: FileStoreID for reference genome fasta
+    :param str ref_fai: FileStoreID for reference genome index file
+    :param str ref_dict: FileStoreID for reference genome sequence dictionary file
+    :param str merge_option: Value for --genotypemergeoption flag (Default: 'UNIQUIFY')
+                            'UNIQUIFY': Multiple variants at a single site are merged into a
+                                        single variant record.
+                            'UNSORTED': Used to merge VCFs from the same sample
+    :return: FileStoreID for merged VCF file
+    :rtype: str
+    """
+    job.fileStore.logToMaster('GATK CombineVariants')
+
+    inputs = {'genome.fa': ref_fasta,
+              'genome.fa.fai': ref_fai,
+              'genome.dict': ref_dict}
+    inputs.update(vcfs)
+
+    work_dir = job.fileStore.getLocalTempDir()
+    for name, file_store_id in inputs.iteritems():
+        job.fileStore.readGlobalFile(file_store_id, os.path.join(work_dir, name))
+
+    command = ['-T', 'CombineVariants',
+               '-R', '/data/genome.fa',
+               '-o', '/data/merged.vcf',
+               '--genotypemergeoption', merge_option]
+
+    for uuid, vcf_id in vcfs.iteritems():
+        command.extend(['--variant', os.path.join('/data', uuid)])
+
+    docker_call(work_dir=work_dir,
+                env={'JAVA_OPTS': '-Djava.io.tmpdir=/data/ -Xmx{}'.format(job.memory)},
+                parameters=command,
+                tool='quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
+                inputs=inputs.keys(),
+                outputs={'merged.vcf': None})
+
+    return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'merged.vcf'))
+
+
+def gatk_combine_gvcfs(job, gvcfs, ref_fasta, ref_fai, ref_dict):
```

Comment:
gatk_combine_gvcfs is actually not necessary because genotype_gvcfs can also combine GVCFs. I will remove it.